src/Microsoft.AspNet.Mvc.TagHelpers/Internal/JavaScriptStringArrayEncoder.cs

```diff
@@ -1,35 +1,38 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.Collections.Generic;
 using System.IO;
 using System.Text.Encodings.Web;
 
 namespace Microsoft.AspNet.Mvc.TagHelpers.Internal
 {
     /// <summary>
-    /// Methods for encoding <see cref="IEnumerable{string}"/> for use as a JavaScript array literal.
+    /// Methods for encoding <c>string[]</c>s for use as a JavaScript array literal.
     /// </summary>
     public static class JavaScriptStringArrayEncoder
     {
         /// <summary>
-        /// Encodes a .NET string array for safe use as a JavaScript array literal, including inline in an HTML file.
+        /// Encodes a <c>string[]</c> for safe use as a JavaScript array literal in many contexts, including
+        /// inline in an HTML file.
         /// </summary>
         public static string Encode(JavaScriptEncoder encoder, string[] values)
+        /// <param name="encoder">
```

Comment:
Looks like these are misplaced. Also why are we documenting types in the `Internal` namespace?